docs/content/en/docs/clustermgmt/cluster-overview.md

```diff
@@ -4,17 +4,35 @@ linkTitle: "Overview"
+description: Overview of EKS Anywhere cluster management
 ---
 
-The content in this page will describe the tools and interfaces available to an administrator after an EKS Anywhere cluster is up and running.
-It will also describe which administrative actions done:
+Once you have an EKS Anywhere cluster up and running, there are a number of operational tasks you may need to perform such as changing cluster configuration, upgrading the cluster Kubernetes version, scaling the cluster, and setting up additional operational software such as ingress/load balancing tools, observability tools, and storage drivers.
 
-* Directly in Kubernetes itself (such as adding nodes with `kubectl`)
-* Through the EKS Anywhere API (such as deleting a cluster with `eksctl`).
-* Through tools which interface with the Kubernetes API (such as [managing a cluster with `terraform`]({{< relref "./cluster-terraform" >}}))
+### Cluster Lifecycle Operations
+
+The tools available for cluster lifecycle operations (create, update, upgrade, scale, delete) vary based on the EKS Anywhere architecture you run. You must use the eksctl CLI for cluster lifecycle operations with standalone clusters and management clusters. If you are running a management / workload cluster architecture, you can use the management cluster to manage one to many downstream workload clusters. With the management cluster architecture, you can use the eksctl CLI, any Kubernetes API-compatible client, or Infrastructure as Code (IAC) tooling such as Terraform and GitOps to manage the lifecycle of workload clusters. For details on the differences between the architecture options, reference the [Architecture page]({{< relref "../concepts/architecture/" >}}).
+
+To perform cluster lifecycle operations for standalone, management, or workload clusters, you modify the EKS Anywhere Cluster specification, which is a Kubernetes Custom Resource for EKS Anywhere clusters. When you modify a field in an existing Cluster specification, EKS Anywhere reconciles the infrastructure and Kubernetes components until they match the new desired state you defined. These operations are asynchronous and you can validate the state of your cluster following the guidance on the [Verify Cluster page]({{< relref "observability/cluster-verify/" >}}). To change a cluster configuration, update the field(s) in the Cluster specification and apply it to your standalone or management cluster. To upgrade the Kubernetes version, update the k`ubernetesVersion` field in the Cluster specification and apply it to your standalone or management cluster. To scale the cluster, update the control plane, etcd, or worker node `count` in the Cluster specification and apply it to the standalone or management cluster, or alternatively use Cluster Autoscaler to automate the scaling process. This operational model facilitates a declarative, intent-based pattern for defining and managing Kubernetes clusters that aligns with modern IAC and GitOps practices.
```

Comment:
```suggestion
To perform cluster lifecycle operations for standalone, management, or workload clusters, you modify the EKS Anywhere Cluster specification, which is a Kubernetes Custom Resource for EKS Anywhere clusters. When you modify a field in an existing Cluster specification, EKS Anywhere reconciles the infrastructure and Kubernetes components until they match the new desired state you defined. These operations are asynchronous and you can validate the state of your cluster following the guidance on the [Verify Cluster page]({{< relref "observability/cluster-verify/" >}}). To change a cluster configuration, update the field(s) in the Cluster specification and apply it to your standalone or management cluster. To upgrade the Kubernetes version, update the `kubernetesVersion` field in the Cluster specification and apply it to your standalone or management cluster. To scale the cluster, update the control plane or worker node `count` in the Cluster specification and apply it to the standalone or management cluster, or alternatively use Cluster Autoscaler to automate the scaling process. This operational model facilitates a declarative, intent-based pattern for defining and managing Kubernetes clusters that aligns with modern IAC and GitOps practices.
```
external etcd count is immutable

Reply:
Really? what happens if you need to scale etcd after cluster creation?

Reply:
can't do that..